challenges/objects-arrays.js

```diff
@@ -97,29 +136,38 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 
 */
 
-const lowerCase = [];
+const lowerCase = zooAnimals.map(element => {
+  return element.animal_name.toLowerCase()
+});
+
 console.log(lowerCase); 
 
 /* Request 3: .filter() 
 
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
-const largerPopulation = [];
+const largerPopulation = zooAnimals.filter(element => {
+  return element.population < 5
+});
+
+
 console.log(largerPopulation);
 
 /* Request 4: .reduce() 
 
 The zoos need to know their total animal population across the United States.  Find the total population from all the zoos using the .reduce() method.
 
 */
-const populationTotal = 0;
+const populationArray = zooAnimals.map(element => {return element.population});
```

Comment:
You could take out the return statement here and make it very streamlined with an `implicit return`:
```
zooAnimals.map(el => el.population);
```